projects/kit/components/pagination/pagination.template.html

```diff
@@ -8,7 +10,7 @@
             appearance="flat"
             icon="tuiIconChevronLeft"
             class="button"
-            [title]="texts[0]"
+            [title]="$any(texts)[0]"
```

Comment:
And here

Reply:
because `text` type is `[string, string] | null`

Reply:
fixed